docs/troubleshoot/app-framework/appfw-tracing.md

```diff
@@ -0,0 +1,49 @@
+# Enabling Tracing
+
+If you need to provide support with tracing information about the App Framework or a particular part of it, or need to debug a program you are developing that uses the App Framework, you can enable a variety of tracing within the Zowe YAML configuration file.
+If you are looking for basic troubleshooting and support, please instead see [Gathering Information for Support or Troubleshooting](../servers/must-gather.md)
+
+# Basic debugging
+
+Within the Zowe YAML file, the value `components.app-server.debug` can be set to `true` to turn on several debug loggers.
+This does not turn on every type of debugging but provides a basic set for debugging for the App Server.
+
+Enabling `components.app-server.debug` is equivalent to setting:
+
+```yaml
+components:
+  app-server:
+    logLevels:
+    - _zsf.*: 5
```

Comment:
This is array syntax, and it should be object syntax, so it should not have that leading hyphen.

Reply:
@nosrednayduj -- I have updated the file. Can you check that the syntax is correct?